plenum/common/exceptions.py

```diff
@@ -296,3 +296,9 @@ class CommonSdkIOException(Exception):
 
 class PoolLedgerTimeoutException(Exception):
     pass
+
+
+class MissingProtocolVersionError(TypeError):
+    def __init__(self, message):
+        super().__init__(
+            message + 'Please update libindy or indy-node to the latest stable version.')
```

Comment:
Do we need a whitespace before `Please`?

Reply:
I've added it after message in _raise_missed_fields

Reply:
Or you think it need to be here?